browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -1064,6 +1064,21 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
       <message name="IDS_BRAVE_NEWS_TITLE" desc="Title for Brave news settings">
         Brave News
       </message>
+      <message name="IDS_CUSTOMIZE_MENU_TITLE" desc="Title for customize menu settings">
+        Customize menu
+      </message>
+      <message name="IDS_CUSTOMIZE_MAIN_MENU_SECTION_TITLE" desc="Title for main menu customization settings">
+        Main menu
+      </message>
+      <message name="IDS_CUSTOMIZE_PAGE_ACTIONS_SECTION_TITLE" desc="Title for page actions customization settings">
+        Page actions
+      </message>
+      <message name="IDS_CUSTOMIZE_PAGE_ACTIONS_SECTION_SUMMARY" desc="Description for page actions customization settings">
+        The following items will only be visible on the menu when supported by the current open webpage.
+      </message>
+        <message name="IDS_WEBAPP" desc="WebApp placeholder used in menu item customization preferences">
+        WebApp
+      </message>
```

Comment:
Is this one being used?

Reply:
Yes it is 

<img width="1080" height="2400" alt="Screenshot_20250915-210420" src="https://github.com/user-attachments/assets/18de0b94-f9f9-46d9-8e95-8f81fc845ff1" />